answerking_app/tests/test_categories.py

```diff
@@ -213,16 +209,16 @@ def test_post_invalid_json_returns_bad_request(self):
         invalid_json_data: str = '{"invalid": }'
         expected_json_error: ErrorMessage = {
             "error": {
-                "message": "Failed data validation",
-                "details": "Invalid JSON in body. Expecting value",
+                "details": "Object could not be created",
```

Comment:
Why is it returning this error? I'd aspect an error from DRF when the json is invalid, isn't this error message the one in views?